app/javascript/styles/mastodon/admin.scss

```diff
@@ -382,6 +383,14 @@ $content-width: 840px;
 
         &.visible {
           display: block;
+          position: fixed;
+          z-index: 10;
+          width: 100%;
+          height: calc(100vh - 56px);
```

Comment:
This causes a bit of a bug on iPhones with notches (e.g. my iPhone 14 Pro), see https://github.com/mastodon/mastodon/issues/21279#issuecomment-1321501142

We could potentially do `height: -webkit-fill-available;` after this line to make it (mostly) work correctly on iPhone? Though, it should only be used on mobile device breakpoints, since it will break the scrollability on macOS Safari.

Not a perfect solution available here, from what I've been able to figure out :/